Code/Tools/ProjectManager/tests/GemCatalogTests.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/UnitTest/TestTypes.h>
+#include <AzTest/Utils.h>
+#include <GemCatalog/GemModel.h>
+
+
+namespace O3DE::ProjectManager
+{
+    class GemCatalogTests 
+        : public ::UnitTest::ScopedAllocatorSetupFixture
+    {
+    public:
+
+        GemCatalogTests() = default;
+    };
+
+    TEST_F(GemCatalogTests, GemCatalog_Displays_But_Does_Not_Add_Dependencies)
```

Comment:
I cyclical dependency test would be really good to ensure this doesn't crash (Doesn't look like it would based off of current implementation though)

Reply:
There are, in fact, many cyclical dependencies already - mostly Atom, though we could add a test later.